teacher.rb

```diff
@@ -1,7 +1,8 @@
 require './person'
+
 class Teacher < Person
-  def initialize(age:, specialization:, name: 'Unknown', parent_permission: true)
-    super(name: name, age: age, parent_permission: parent_permission)
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] again as the previous CR mentioned this will not work, please remove the colons, if you have any questions or you don't understand the requirements please leave a message in the PR.